operational/fnms/schedule_fnms_reports/README.md

```diff
@@ -1,46 +1,66 @@
-## Schedule FlexNet Manager report
+# Schedule FlexNet Manager report
+
+## What it does
 
-### What it does
 This policy will run a FlexNet Manager report (Custom view) and send the result via email.
```

Comment:
You should include some details about the Flexnet Manager System on-premise needs to be internet accessible.  and provide details about wstunnel.